scripts/hostapd/95-wlan-hostap.link

```diff
@@ -0,0 +1,15 @@
+# make the wlan devices for hostap match "*hostap.link"
+# By default, all wlan interfaces will be access points
+[Match]
+Type=wlan
+Driver=xradio_wlan 8189fs
+
+[Link]
+NamePolicy=kernel database
+MACAddressPolicy=persistent
+
+# this section gets appended to the hostap conf, overwriting any default
+# values from the template
+[hostapd]
+ssid=##SSID##
+wpa_passphrase=bbbbbbbb
```

Comment:
Where are we setting the password passed in from install script?

Reply:
Ported code for asking for a password on install